WordPress/gutenberg
# RNMobile: Fix slider and stepper text inputs in dark mode

## Description

See https://github.com/wordpress-mobile/gutenberg-mobile/pull/3724

This fixes a bug where sliders were not displaying properly in dark mode on iOS. Specifically, the text input field that's shown next to sliders (e.g. the Spacer block's height slider control) were showing dark text in dark mode. 

This issue first appeared in https://github.com/WordPress/gutenberg/releases/tag/rnmobile/1.56.0 which was the first release after the RN 0.64 update (https://github.com/WordPress/gutenberg/pull/29118).

It appears that RN used to use white as the default text input text color when dark mode was enabled, and we relied that for some time. In RN 0.64 however, this default behavior changed and now black is the default. This means we must now explicitly define text colors for text inputs and that's what this PR does.

Note: This also tweaks the Buttons block border radius test in https://github.com/WordPress/gutenberg/pull/33211 to make it pass given the changes in this PR.

## How has this been tested?
This fix is meant to address  all `RangeTextInput` components. These are found in sliders:
- Spacer block: height
- Button block: border radius
- Cover block: focal point position, opacity
- Column block: width
- Latest posts block: number of items

And steppers:
- Cover block: minimum height

### To test
1. Run the app on iOS and verify that the input fields next to sliders and steppers look good in both dark and light mode
2. Run the app on Android and verify that there are no changes

## Screenshots 

The below screenshots how how the fix looks on sliders. The same applies for steppers.

| Before | After |
| - | - |
| <img width="564" alt="Screen Shot 2021-07-13 at 17 40 57" src="https://user-images.githubusercontent.com/1898325/125529208-943bce27-4b8f-4681-9067-55cbc10f3a08.png"> | <img width="564" alt="Screen Shot 2021-07-13 at 17 41 05" src="https://user-images.githubusercontent.com/1898325/125529192-f4fc4bc0-c8d3-40b1-b982-3b8ac126c05b.png"> |


## Types of changes
- Bug fix

## Checklist:
